src/instruments/buildSrc/igniter/RollupTask.mjs

```diff
@@ -0,0 +1,15 @@
+import { GenericTask } from '@flybywiresim/igniter';
+import { promisify } from 'util';
+import { exec } from 'child_process';
+
+export class RollupTask extends GenericTask {
+    constructor(
+        key,
+        targetToBuild,
+        hashFolders,
+    ) {
+        super(key, async () => {
+            await promisify(exec)(`node src/instruments/buildSrc/igniter/RollupTaskWorker.mjs ${targetToBuild}`);
```

Comment:
Why not just have the task be the code in `RollupTaskWorker` as opposed to running the code using node? (i.e. merge the two files)

Reply:
It wouldn't run asynchronously then would it ?

Reply:
Good catch, yes. You'd have to use [worker threads](https://nodejs.org/api/worker_threads.html). Up to you if you wanna keep exec. I'm happy either way